src/3d/CMakeLists.txt

```diff
@@ -110,6 +111,9 @@ SET(QGIS_3D_HDRS
   chunks/qgschunkloader_p.h
   chunks/qgschunkqueuejob_p.h
   mesh/qgsmeshterraingenerator.h
+  qgs3dsceneexporter.h
+  qgs3dmapexportsettings.cpp
```

Comment:
cpp file within headers section?

Reply:
Sorry I added it by mistake and forgot it there :facepalm: